router/pkg/otel/attributes.go

```diff
@@ -32,6 +32,7 @@ const (
 	WgAcquireResolverWaitTimeMs        = attribute.Key("wg.engine.resolver.wait_time_ms")
 	WgNormalizationCacheHit            = attribute.Key("wg.engine.normalization_cache_hit")
 	WgValidationCacheHit               = attribute.Key("wg.engine.validation_cache_hit")
+	WgVariablesValidationSkipped       = attribute.Key("wg.engine.variables_validation_skipped")
```

Comment:
Do you think this belongs to the `resolver` namespace? See `wg.engine.resolver.wait_time_ms`